iota/__init__.py

```diff
@@ -29,17 +29,17 @@
 
 
 # Activate codecs.
-from .codecs import *
+from .codecs import *  # noqa
 
 # Make some imports accessible from the top level of the package.
 # Note that order is important, to prevent circular imports.
-from .types import *
-from .transaction import *
-from .adapter import *
-from .api import *
-from .trits import *
+from .types import *  # noqa
+from .transaction import *  # noqa
+from .adapter import *  # noqa
+from .api import *  # noqa
+from .trits import *  # noqa
 
 # :see: http://stackoverflow.com/a/2073599/
-from pkg_resources import require
+from pkg_resources import require  # noqa
```

Comment:
❔ I could see why the wildcard imports would need a `# noqa`, but does this import need one as well?

Reply:
`require` is not used. As I just read this is a warning of flake, not a PEP8 violation. But, if we include flake8 into travisCI this would be nice to get rid off. One way of handling this would be to explicitly export imported classes via [`__all__`](https://stackoverflow.com/questions/31079047/python-pep8-class-in-init-imported-but-not-used)

Reply:
Gotcha.  Hm, that's odd.  It does get used, but then it gets removed via `del` later on (I don't want it included in the `iota` namespace).

Eh, maybe the `# noqa` here is fine.